packages/mutation-testing-elements/package.json

```diff
@@ -26,5 +26,9 @@
   "devDependencies": {
     "mutation-testing-metrics": "^1.2.2",
     "mutation-testing-report-schema": "^1.2.0"
+  },
+  "dependencies": {
+    "@types/prismjs": "^1.16.0",
```

Comment:
`@Types` are always dev-dependencies

Reply:
Added in 1007d2d7a09e05980e7b5232d3ef014075ae165d